clients/src/main/java/org/apache/kafka/common/security/oauthbearer/secured/package-info.java

```diff
@@ -16,6 +16,6 @@
  */
 /**
  * This package is deprecated.
- * @deprecated See {@link org.apache.kafka.common.security.oauthbearer}
+ * See {@link org.apache.kafka.common.security.oauthbearer}
```

Comment:
This was triggering the following warning: `Tag @deprecated cannot be used in package documentation.`